twa

```diff
@@ -80,6 +80,33 @@ function get_field {
       tolower(\$1) ~ /^${field}/ { print substr(\$0, index(\$0, \$2)) }" | tr -d '\r\n'
 }
 
+function get_cookies {
+  cookieheader="set-cookie"
+
+  verbose "Extracting cookies from headers"
+
+  awk "tolower(\$0) ~ /^${cookieheader}:/ { print substr(\$0, index(\$0, \$2)) }"
```

Comment:
Since this function will only ever get cookies, removing the `cookieheader` variable here is fine:

```
awk "tolower(\$0) ~ /^set-cookie:/ { print substr(\$0, index(\$0, \$2)) }"
```